packer/builder/azure/arm/step_ip_address.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Microsoft Open Technologies, Inc.
+// All Rights Reserved.
+// Licensed under the Apache License, Version 2.0.
+// See License.txt in the project root for license information.
+
+package arm
+
+import (
+	"fmt"
+
+	"github.com/Azure/packer-azure/packer/builder/azure/common/constants"
+	"github.com/mitchellh/multistep"
+	"github.com/mitchellh/packer/packer"
+)
+
+type StepIPAddress struct {
```

Comment:
Maybe add a verb to the name so that we know what this step does with the IPAddress?